source/unified-test-format/tests/invalid/clientEncryptionOpts-additionalProperties.yml

```diff
@@ -2,6 +2,9 @@ description: clientEncryptionOpts-additionalProperties
 
 schemaVersion: "1.8"
 
+runOnRequirements:
+  - csfle: true
```

Comment:
The `invalid` tests are only testing JSON schema validation and are never actually loaded by the test runner. I don't think they need this `runOnRequirement` at all.